src/SDKs/NetApp/NetApp.Tests/NetApp.Tests.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>NetApp.Tests</PackageId>
     <Description>NetApp.Tests Class Library</Description>
     <AssemblyName>NetApp.Tests</AssemblyName>
-    <Version>1.0.0-preview</Version>
```

Comment:
Test projects are never published. Please set this to `1.0.0` and do not modify henceforth

Reply:
ok. Done.